server/src/data-layer/models/firebase.ts

```diff
@@ -166,4 +186,9 @@ export interface EventSlot {
    * @example 30
    */
   max_occupancy?: number
+  /**
+   * Subcollection of EventReservation interface.
+   * Holds the members that made a reservation to this event
+   */
+  reservations: EventReservation[]
```

Comment:
don't need this refer to what subcollections are in firebase https://firebase.google.com/docs/firestore/data-model#subcollections